containers/test-apps/jwks-server/src/jwks_server/handler.clj

```diff
@@ -19,14 +19,17 @@
             [clojure.data.json :as json]
             [clojure.tools.logging :as log]
             [clojure.string :as str]
+            [clojure.walk :as walk]
             [jwks-server.config :as config]
-            [plumbing.core :as pc]
             [ring.middleware.params :as ring-params]
-            [ring.util.request :as ring-request]))
+            [ring.util.request :as ring-request])
+  (:import (java.security MessageDigest)
+           (java.util Base64 UUID)))
 
 (defn prepare-response
   "Prepares and returns a standard response"
   ([{:keys [query-string request-method uri]} message status]
+   (log/info message)
```

Comment:
Should we add some context, like "Returning response" with a map that contains message, status, etc.?

Reply:
Logs look something like this:
```
2020-05-29 11:48:42 INFO  jwks-server.handler - received :get request as path /authorize                                                                                                                                                                                     
2020-05-29 11:48:42 INFO  jwks-server.handler - request headers: {user-agent waiter-test/oidc-auth-20200513.http1, x-cid test-wattoar-3815432a40fa0-7ce41cc6660448f6, host 127.0.0.1:8080, accept-encoding gzip}                                                             
2020-05-29 11:48:42 INFO  jwks-server.handler - inserted access code entry for 33c53cb0-c5fa-4d6e-8b80-a74954dad5bd  
2020-05-29 11:48:42 INFO  jwks-server.handler - response status: 301                                           
2020-05-29 11:48:42 INFO  jwks-server.handler - response headers: {location https://wattoar986623473682231.127.0.0.1/oidc/v1/callback?code=33c53cb0-c5fa-4d6e-8b80-a74954dad5bd&state=TlBZAkRbY/o/lzq8mX0t2w6LEmE X2dA8LoGuTTl6HVypsW7O0Rw1JXYxUQ3TfRRV91nxnbjHPyfnp9nEq9aevV
Y5O6YssVpBogegZXqy6Gh7Kkv/MxdR7ZyzAg3MJJvPyVUvg==, server jwks-server}       
```